charts/datadog/templates/cluster-agent-rbac.yaml

```
@@ -238,6 +238,17 @@ rules:
   resources: ["secrets"]
   verbs: ["get"]
 {{- end }}
+{{- if .Values.datadog.helmCheck.enabled}}
```

IMO, it should only be added if clustercheck is not enabled

```suggestion
{{- if and .Values.datadog.helmCheck.enabled (not .Values.datadog.clusterChecks.enabled}}
```

---

Yes, that would be better :+1: